src/bin/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
     endif()
     SET_TARGET_PROPERTIES(msquic
         PROPERTIES LINK_FLAGS "/DEF:\"${CMAKE_CURRENT_SOURCE_DIR}/winuser/msquic.def\"")
-else()
+elseif (QUIC_PLATFORM STREQUAL "linux")
     SET_TARGET_PROPERTIES(msquic
         PROPERTIES LINK_FLAGS "-Wl,--version-script=\"${CMAKE_CURRENT_SOURCE_DIR}/linux/exports.txt\"")
 endif()
```

Comment:
Theres a macOS equivalent of this setup. 

The file is just a single line per symbol, prefixed with `_`. Place that file in darwin/exports.txt. That file is then passed to the linker with `-exported_symbols_list ${CMAKE_CURRENT_SOURCE_DIR}/darwin/exports.txt`

Reply:
@thadhouse would you mind pushing your suggested change to this branch?

Reply:
I will once it builds successfully. I don't want to interfere with getting that working.